com.ibm.streamsx.topology/bin/spl-python-extract.py

```diff
@@ -106,13 +114,101 @@ def copyGlobalizationResources():
                 os.makedirs(dstDir)
             except OSError as e:
                 if (e.errno == 17) and (os.path.isdir(dstDir)):
-                    print(" dir exists")
+                    if cmd_args.verbose:
+                        print("Directory", dstDir, "exists")
                 else:
                     raise
             srcFile = os.path.join(srcDir, "TopologySplpyResource.xlf")
             if os.path.isfile(srcFile):
                 res = shutil.copy2(srcFile, dstDir)
-                print("Written: " + res)
+                languageList.append(dirName)
+                if cmd_args.verbose:
+                    print("Written: " + res)
+    return languageList
+
+_INFO_XML_TEMPLATE="""<?xml version="1.0" encoding="UTF-8"?>
+<toolkitInfoModel
+  xmlns="http://www.ibm.com/xmlns/prod/streams/spl/toolkitInfo"
+  xmlns:common="http://www.ibm.com/xmlns/prod/streams/spl/common">
+  <identity>
+    <name>__SPLPY_TOOLKIT_NAME__</name>
+    <description>Automatic generated toolkit description file.</description>
+    <version>1.0.0</version>
+    <requiredProductVersion>4.1.0.0</requiredProductVersion>
```

Comment:
Is there a reason to limit this to 4.1? The current toolkit supports 4.0.1 or later.